distribution/sdk/sample-nsili-client/src/main/java/org/codice/alliance/nsili/client/SampleNsiliClient.java

```diff
@@ -492,180 +530,156 @@ public void testStandingQueryMgr(POA poa, Query query) throws Exception {
-                String thumbFile = productID + "-thumbnail" + ".jpg";
-                LOGGER.info("Getting thumbnail for : {}.  Filenamne : {}", productID, thumbFile);
-                FileLocation thumbnailLoc = new FileLocation("user",
-                        "pass",
-                        "localhost",
-                        "/nsili/file",
-                        thumbFile);
-                GetRelatedFilesRequest request = productMgr.get_related_files(new Product[] {
-                        product}, thumbnailLoc, "THUMBNAIL", getRelatedFileProps);
-                request.set_user_info("Alliance");
-                NameListHolder locations = new NameListHolder();
-                request.complete(locations);
-
-                for (String location : locations.value) {
-                    LOGGER.info("\t Stored File: {}", location);
-                }
-            } catch (Exception e) {
-                LOGGER.error("Unable to get product thumbnail.",
-                        NsilCorbaExceptionUtil.getExceptionDetails(e));
-            }
+            LOGGER.info("Getting related file types for : {}", productID);
+            get_related_file_types(product);
 
-            try {
-                String productID = getProductID(dags[0]);
-                LOGGER.info("Getting overview for : {}", productID);
-                FileLocation thumbnailLoc = new FileLocation("user",
-                        "pass",
-                        "localhost",
-                        "/tmp/files",
-                        productID + "-overview" + ".jpg");
-                GetRelatedFilesRequest request = productMgr.get_related_files(new Product[] {
-                        product}, thumbnailLoc, "OVERVIEW", getRelatedFileProps);
-                request.set_user_info("Alliance");
-                NameListHolder locations = new NameListHolder();
-                request.complete(locations);
-            } catch (Exception e) {
-                LOGGER.error("Unable to get product overview: {}",
-                        NsilCorbaExceptionUtil.getExceptionDetails(e),
-                        e);
-            }
+            LOGGER.info("Getting thumbnail for : {}", productID);
+            get_related_files(product);
         } else {
-            LOGGER.warn("Unable to test ProductMgr as it is not set");
+            LOGGER.warn("ProductMgr is not initialized, unable to test");
         }
     }
 
-    public void get_parameters(Product product) throws Exception {
-        if (productMgr != null) {
-            LOGGER.info("Sending Get Parameters Request to Server...");
+    public Product getProductFromDag(DAG dag) {
+        Product product = ProductHelper.extract(dag.nodes[0].value);
+        LOGGER.info("Product: {}", product.toString());
+        LOGGER.info("Product ID: {}", getProductIdFromDag(dag));
+        return product;
+    }
+
+    public DAG get_parameters(DAG dag) throws InvalidInputParameter, SystemFault, ProcessingFault {
```

Comment:
Updated.

I think the motivation behind the `_`s was to match the corresponding NSILI manager methods that will be called.